### PR TITLE
Introducing options.cloud

### PR DESCRIPTION
**Pull request by olegbespalov**
## What?

This PR documents the `options.cloud` introduced in https://github.com/grafana/k6/pull/3348

I'm unsure if we should still promote ext; that's why I replaced it.

## Checklist

- [ ] I have used a meaningful title for the PR.
- [ ] I have described the changes I've made in the "What?" section above.
- [ ] I have performed a self-review of my changes.
- [ ] I have run the `npm start` command locally and verified that the changes look good.
- [ ] I have made my changes in the `docs/sources/next` folder of the documentation.

## Related PR(s)/Issue(s)

https://github.com/grafana/k6/pull/3348

### CODE REVIEW

**Review by mstoykov — APPROVED** [review]

**Review by joanlopez — APPROVED** [review]

File: docs/sources/next/using-k6/k6-options/reference.md [hunk 1]
```diff
@@ -352,13 +352,13 @@ export const options = {
 
 {{< /code >}}
 
-## Extension options
+## Cloud options
 
 An object used to set configuration options for cloud parameters and third-party collectors, like plugins. For more information about available parameters, refer to [Cloud options](https://grafana.com/docs/grafana-cloud/k6/author-run/cloud-scripting-extras/cloud-options/).
```
Comment (heitortsergent) [inline docs/sources/next/using-k6/k6-options/reference.md:357]: @olegbespalov, I saw you removed "third-party collectors" from the table above. Do we want to remove it from here as well?

**Review by heitortsergent — APPROVED** [review]
Left a small comment, overall it looks good! I'll take a look at the Grafana Cloud k6 docs as well and see if there are any updates we need to make there.

**Comment by oleiade** [comment]
As Oleg is away this week, I'll merge this to avoid blocking the release process 🙇‍♂️

File: docs/sources/next/using-k6/k6-options/reference.md (hunk 1 above)
Comment (olegbespalov) [reply docs/sources/next/using-k6/k6-options/reference.md:357]: Yep, it makes sense to remove this from here, good catch :+1:  I'll open a PR

**Pull request merged**